code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -42,6 +43,33 @@
 	result_amount = 2
 	strengthdiv = 10
 
+/datum/chemical_reaction/reagent_explosion/potassium_explosion/holyboom
+	name = "Holy Explosion"
+	id = "holyboom"
+	required_reagents = list("holywater" = 1, "potassium" = 1)
+
+/datum/chemical_reaction/reagent_explosion/potassium_explosion/holyboom/on_reaction(datum/reagents/holder, created_volume)
+	if(created_volume >= 150)
+		playsound(get_turf(holder.my_atom), 'sound/effects/pray.ogg', 80, 0, round(created_volume/48))
+		strengthdiv = 8
+		for(var/mob/living/simple_animal/revenant/R in get_hearers_in_view(7,get_turf(holder.my_atom)))
+			var/diety = ticker.Bible_deity_name
+			if(!ticker.Bible_deity_name)
+				diety = "Christ"
+			R << "<span class='userdanger'>The power of [diety] compels you!</span>"
+			R.stun(20)
+			R.reveal(100)
+		sleep(20)
```

Comment:
That's the tradeoff for having a bigger boom. (200u yields 1,3,7 as opposed to 0,3,6)
It's not meant to be too practical, but something fun.